Description: adjust ejabberdctl.cfg.example to create a PID file
 ejabberd's default control config doesn't create a PID file upon start,
 so change it
Author: Philipp Huebner <debalance@debian.org>

Index: ejabberd/ejabberdctl.cfg.example
===================================================================
--- ejabberd.orig/ejabberdctl.cfg.example
+++ ejabberd/ejabberdctl.cfg.example
@@ -134,7 +134,7 @@
 #
 # Default: ejabberd@localhost
 #
-#ERLANG_NODE=ejabberd@localhost
+#ERLANG_NODE=ejabberd
 
 #.
 #' EJABBERD_PID_PATH: ejabberd PID file
@@ -146,7 +146,7 @@
 #
 # Default: don't write PID file
 #
-#EJABBERD_PID_PATH=/var/run/ejabberd/ejabberd.pid
+EJABBERD_PID_PATH=/run/ejabberd/ejabberd.pid
 
 #.
 #' EJABBERD_CONFIG_PATH: ejabberd configuration file
